deutsche-bank-adapter/src/main/java/de/adorsys/xs2a/gateway/service/impl/DeutscheBankObjectMapper.java

```diff
@@ -17,11 +17,37 @@
 package de.adorsys.xs2a.gateway.service.impl;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
 
 class DeutscheBankObjectMapper extends ObjectMapper {
 
     DeutscheBankObjectMapper() {
+        registerModule(new JavaTimeModule());
         setSerializationInclusion(JsonInclude.Include.NON_NULL);
+        disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+    }
+
+    <T> String toString(T body) {
```

Comment:
I would like to discuss, whether it is OK for us, that the user of `DeutscheBankObjectMapper` can call the `writeValueAsString(...)` and `readValue(...)` methods of `ObjectMapper` class, as we don't override them, but provide the new methods instead?

Reply:
Also, I'd propose to discuss using HAS-A instead of IS-A for this case.

Reply:
If we override them we should catch all exceptions from object mapper methods signature

Reply:
Sorry, but no, we shouldn't
![image](https://user-images.githubusercontent.com/24653053/55890000-a01fe400-5bba-11e9-8f1b-4979f452d4ad.png)

Reply:
What do you mean?
![image](https://user-images.githubusercontent.com/24653053/55890201-f2610500-5bba-11e9-8903-e3d598927277.png)

Reply:
But I'd better go with HAS-A approach to have the full control on custom object mapper API. What do you think?